security/pom.xml

```
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.devonfw.tools.IDEasy.dev</groupId>
+        <artifactId>ide</artifactId>
+        <version>dev-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.example</groupId>
+    <artifactId>security</artifactId>
+
+    <properties>
+        <java.version>17</java.version>
+        <owasp.version>9.0.9</owasp.version>
+    </properties>
+    <dependencies>
+        <!-- Other configurations and properties -->
+        <dependency>
+            <groupId>org.owasp</groupId>
+            <artifactId>dependency-check-core</artifactId>
+            <version>${owasp.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.devonfw.tools.IDEasy</groupId>
+            <artifactId>ide-cli</artifactId>
+            <version>2024.02.001-alpha-SNAPSHOT</version>
```

```suggestion
            <version>${revision}</version>
```

We can even define `<dependencyManagement>` in parent `pom.xml` and entirely omit the dependency versions here.

---

Done.